### ISSUE TITLE
Many new logo request

### ISSUE THREAD

**Issue by gokaygurcan** [gilbarbara/logos#20]
Hey, 
some people already asked for Apple, Microsoft and Jira but on top them, I also would like to see these ones:

- [ ] [Adobe](http://www.adobe.com/legal/permissions/icons-web-logos.html)
- [ ] [Adyen](https://avatars.githubusercontent.com/u/9713612?v=3)
- [ ] [Adyen v2](http://www.freelogovectors.net/wp-content/uploads/2013/04/adyen-logo.jpg)
- [x] [Atlassian products](https://design.atlassian.com/2.0/how-we-design/resources/)
- [x] [Basho / Riak](http://basho.com/design-assets-hub/)
- [x] [Dart](https://www.dartlang.org/logos/)
- [x] [dotNet Foundation](https://github.com/dotnet/swag/tree/master/logo)
- [x] [Eclipse](https://eclipse.org/artwork/)
- [x] [Electron](https://camo.githubusercontent.com/5dd01312b30468423cb45b582b83773f5a9019bb/687474703a2f2f656c656374726f6e2e61746f6d2e696f2f696d616765732f656c656374726f6e2d6c6f676f2e737667)
- [x] [Erlang](https://c2.staticflickr.com/4/3095/3172415460_ce2a85308d.jpg) ([or this](http://www.erlang-factory.com/upload/peopleimage/SFBay2012_RobertVirding/Erlang-logo-4-col.png))
- [x] [Famous](https://avatars1.githubusercontent.com/u/838470?v=3)
- [x] [Ghost](https://ghost.org/about/logos/)
- [x] [KeenIO](https://keen.io/blog/43496487388/letters-and-symbols-how-i-designed-the-keen-io-logo)
- [x] [Kitematic](https://cloud.githubusercontent.com/assets/251292/5269258/1b229c3c-7a2f-11e4-96f1-e7baf3c86d73.png)
- [ ] [Koa](https://camo.githubusercontent.com/674563115c4e0d4e5d99440b916952ad795c498e/68747470733a2f2f646c2e64726f70626f7875736572636f6e74656e742e636f6d2f752f363339363931332f6b6f612f6c6f676f2e706e67)
- [x] [MailChimp](http://mailchimp.com/about/brand-assets/)
- [ ] [MailGun](https://github.com/mailgun/media)
- [x] [Mandrill](http://mandrill.com/brand-assets/)
- [ ] [Microsoft](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/logo.aspx)
- [x] [Mono](http://www.mono-project.com/docs/about-mono/logos/)
- [x] [NetBeans](https://netbeans.org/community/teams/evangelism/collateral.html?print=yes)
- [ ] [Open Hack](http://openhack.github.io/images/logo-green.png)
- [ ] [Open Source](http://opensource.org/logo-usage-guidelines)
- [x] [Raspberry Pi](https://www.raspberrypi.org/trademark-rules/)
- [ ] [RethinkDB](https://platzi.com/blog/content/images/2015/03/rethinkdb.png)
- [x] [Segment](https://github.com/segmentio/logo)
- [x] [Sinatra](https://github.com/sinatra/resources/tree/master/logo)
- [x] [Subversion](https://svn.apache.org/repos/asf/subversion/svn-logos/logo.html)
- [ ] [underscore.js](http://underscorejs.org/docs/images/underscore.png)
- [ ] [vim](http://www.vim.org/logos.php)
- [x] [Visual Studio](http://www.daoudisamir.com/wp-content/uploads/2015/01/visual-studio-2013-logo.png)
- [x] [WordPress](https://wordpress.org/about/logos/)

I tried to find useful resources, mostly from their official sites. I hope that help.

Thanks in advance and good luck ;)

**Comment by lukeholder** [comment]
Hey @gokaygurcan maybe make these into github checkboxes or seperate issues?
https://github.com/blog/1375%0A-task-lists-in-gfm-issues-pulls-comments

**Comment by gokaygurcan** [comment]
Hello @lukeholder ,
I updated the comment above and decided to put checkboxes instead of many more issues that probably will be hard to follow after some point. 
Thanks a lot for your suggestion btw.

**Comment by CentaurWarchief** [comment]
@gokaygurcan +RabbitMQ +InfluxDB!

**Comment by gilbarbara** [comment]
added some in a6df8c5feb81efde698dc5f200c707c1527dd442 and  014bfa17d2304ab9febf6b45a268f85aa0720320

I won't add Adobe, Microsoft or any other monolithic companies. And some other logos are fugly!

**Comment by gilbarbara** [comment]
New additions 67287ea9e3214975c86bc0a61aaf65aeca8e8372

**Comment by CentaurWarchief** [comment]
@gilbarbara Any luck with Rabbit & Influx? Also, you could add: https://github.com/facebook/graphql/blob/master/resources/GraphQL%20Logo.svg.

**Comment by gilbarbara** [comment]
@CentaurWarchief Well, Hello!

Please add your own issues separately and I'll search for the logos when it's possible.
And remember that this is a personal project and there's no minimum time frame to fulfillment.

**Comment by CentaurWarchief** [comment]
Roger that.

**Comment by gilbarbara** [comment]
A few more here 9926b3d01fb9abffa26e95cebaf441285581f224

**Comment by gokaygurcan** [comment]
Thanks a lot for all! Good work!
